scripts/tasks.sh

```diff
@@ -490,32 +490,32 @@ spec:
   template:
     spec:
       tolerations:
-        - key: node-role.kubernetes.io/master
+        - key: node-role.kubernetes.io/control-plane
```

Comment:
Rather than remove the existing `node-role.kubernetes.io/master` toleration can we append `node-role.kubernetes.io/control-plane` as an additional toleration?